portfolio/src/main/webapp/comments.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="comments_script.js"></script>
   </head>
+  <!-- get a max of 20 comments initially, edited here -->
```

Comment:
ahh I see, maybe also add this comment to the 
if(co == null){
      comments_count = 20;
    }
logic in the java code?